src/libraries/System.Data.Odbc/src/CompatibilitySuppressions.xml

```diff
@@ -147,84 +147,203 @@
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Data.Odbc.ODBC32</Target>
-    <Left>runtimes/freebsd/lib/netcoreapp2.0/System.Data.Odbc.dll</Left>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
     <Right>lib/netstandard2.0/System.Data.Odbc.dll</Right>
     <IsBaselineSuppression>true</IsBaselineSuppression>
   </Suppression>
   <Suppression>
     <DiagnosticId>CP0002</DiagnosticId>
     <Target>M:System.Data.Odbc.OdbcParameter.get_Offset</Target>
-    <Left>runtimes/freebsd/lib/netcoreapp2.0/System.Data.Odbc.dll</Left>
+    <Left>runtimes/freebsd/lib/net6.0/System.Data.Odbc.dll</Left>
```

Comment:
I need to check these runtime net6.0 assets vs lib assets.

Reply:
Ok, so I was able to figure these out. The package doesn't have a `ref` folder, and this is comparing `runtimes/*` as the contract, VS the ridless `lib` (the compile asset), and it seems like the ref is not exposing these APIs. It seems like that is by design. As we even have this: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Data.Odbc/src/MatchingRefApiCompatBaseline.txt